internal/cli/install.go

```diff
@@ -494,15 +498,20 @@ func installRunner(
 		AuthToken:     resp.Token,
 	}
 
+	// We set the ID to be "init" since it is the initial static runner
+	id := "static"
```

Comment:
Ahh yeah, I saw your notice about not adding runner here. I'd make a comment that it's expected from the supported platforms that it adds a `-runner` to the name later.